test/integration/run_int_test.go

```diff
@@ -281,6 +281,32 @@ func (suite *RunIntegrationTestSuite) TestRun_BadLanguage() {
 	cp.Expect("The language for this script is not supported", 5*time.Second)
 }
 
+func (suite *RunIntegrationTestSuite) TestRun_Perl_Variable() {
+	if runtime.GOOS == "windows" {
+		suite.T().Skip("Testing exec of Perl with variables is not applicable on Windows")
```

Comment:
Why not?

Reply:
Oh cause it's running CMD. IIRC we do allow running bash on Windows. But probably not worth it for this test.

Reply:
Windows doesn't interpret '$' specially, so this isn't an issue. I'll go ahead and enable it anyway.

Reply:
Scratch that, Windows just doesn't like the command syntax and this really isn't applicable.